mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAOptimizer.scala

```diff
@@ -451,10 +451,10 @@ final class OnlineLDAOptimizer extends LDAOptimizer {
       }
       Iterator((stat, gammaPart))
     }
-    val statsSum: BDM[Double] = stats.map(_._1).reduce(_ += _)
```

Comment:
It would be better to modify the first argument, which treeAggregate should support.  (Actually I noticed treeAggregate does not say it supports it in the docs, but it should be OK to assume.  I just created [https://issues.apache.org/jira/browse/SPARK-14408] for that.)